2 - Triplet sampling (Random).ipynb

```diff
@@ -134,6 +134,14 @@
     "interactions_df[\"user_id\"] = interactions_df[\"user_id\"].map(user_id2index)\n",
     "print(f\">> Mapping applied, ({n_missing_ids} values in 'user_id2index')\")\n",
     "\n",
+    "# Mark interactions used for evaluation procedure if needed\n",
+    "if \"evaluation\" not in interactions_df:\n",
+    "    print(\"\\nApply evaluation split...\")\n",
+    "    interactions_df = mark_evaluation_rows(interactions_df)\n",
+    "    # Check if new column exists and has boolean dtype\n",
+    "    assert interactions_df[\"evaluation\"].dtype.name == \"bool\"\n",
+    "    print(f\">> Interactions: {interactions_df.shape}\")\n",
+    "\n",
```

Comment:
I forgot, why was this needed here?

Reply:
I just noticed, the code was not present in this repository but it was in mine, right?

Reply:
Correctly.